database/init-db.sql

```diff
@@ -0,0 +1,82 @@
+-- Description: This script contains the SQL commands to create the database schema and insert example data.
+
+
+-- CREATING TABLES
+
+-- Create Status table
+CREATE TABLE IF NOT EXISTS Status (
+    Id SERIAL PRIMARY KEY,
+    Title VARCHAR(255) NOT NULL,
+    Style VARCHAR(255)
+);
+
+-- Create Tag table
+CREATE TABLE IF NOT EXISTS Tag (
+    Id SERIAL PRIMARY KEY,
+    Name VARCHAR(255) NOT NULL,
+    Color VARCHAR(255)
+);
+
+-- Create ActivityType table
+CREATE TABLE IF NOT EXISTS ActivityType (
+    Id SERIAL PRIMARY KEY,
+    Name VARCHAR(255) NOT NULL
+);
+
+-- Create Activity table
+CREATE TABLE IF NOT EXISTS Activity (
+    Id SERIAL PRIMARY KEY,
+    Title VARCHAR(255) NOT NULL,
+    Description TEXT,
+    Url VARCHAR(255),
+    StartDate TIMESTAMP,
+    EndDate TIMESTAMP,
+    Status INTEGER REFERENCES Status(Id),
+    Tags TEXT, -- Consider using an association table for a normalized design
+    ActivityType INTEGER REFERENCES ActivityType(Id)
+);
+
+-- Create Task table
+CREATE TABLE IF NOT EXISTS Task (
+    Id SERIAL PRIMARY KEY,
+    Name VARCHAR(255) NOT NULL,
+    Content TEXT,
+    StartDate TIMESTAMP,
+    EndDate TIMESTAMP,
+    Tags TEXT, -- Consider using an association table for a normalized design
+    Status INTEGER REFERENCES Status(Id),
+    ActivityId INTEGER REFERENCES Activity(Id)
+);
+
```

Comment:
```suggestion

CREATE TABLE IF NOT EXISTS ActivityTag (
    ActivityId INTEGER REFERENCES Activity(Id),
    TagId INTEGER REFERENCES Tag(Id),
    PRIMARY KEY (ActivityId, TagId)
);

CREATE TABLE IF NOT EXISTS TaskTag (
    TaskId INTEGER REFERENCES Task(Id),
    TagId INTEGER REFERENCES Tag(Id),
    PRIMARY KEY (TaskId, TagId)
);
```
and add here two tables like this ones

Reply:
fixed